### PR TITLE
Added in round statistics

### PR DESCRIPTION
**Pull request by FL3SH**
Now you can view your average time during rounds including practice mode.
In practice mode real round count is shown instead of 1 as before.

### CODE REVIEW

**Comment by brandoncc** [comment]
@FL3SH nicely done :-)

Instead of having the `Math.abs` hack, I wonder if we would be better to introduce a round limit number for each difficulty...and we could just set noob to have a really high limit? That would replace the limit of 10 rounds. For that matter, maybe a "round count" option on the menu would be a better route.

Thoughts @ThePrimeagen?

**Comment by FL3SH** [comment]
I thought about sth similar, but I really don't like to see round `2/999999` in practice mode, so if `round limit` will be introduced displayed values must be different, like `2/∞`

**Comment by brandoncc** [comment]
I would be fine with that. We could use [Inifinity](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity). Then in the display, we can show the right symbol.

**Comment by FL3SH** [comment]
Nice, I like it, even just `2/Infinity` is fine, but I am not sure if is worth to change more than a few line of code before migration to Lua

**Comment by brandoncc** [comment]
I think Prime is second guessing the change to Lua, but I am not sure about that. Only @ThePrimeagen knows :-)

I agree that if that rewrite is coming, a hack might be okay.

**Comment by ThePrimeagen** [comment]
Sorry.  I have been... distant!

I have been away trying to focus on the family things.

File: src/game/base.ts [hunk 1]
```diff
@@ -180,8 +180,11 @@ export class Game implements IGame {
 
     public nextRoundNumber() {
         if (this.difficulty === GameDifficulty.Noob) {
-            console.log("base - incrementRoundNumber - noob");
-            return 1;
+            console.log(
+                "base - decrementRoundNumber#noob - ",
+                this.state.currentCount,
+            );
+            return this.state.currentCount - 1;
```
Comment (ThePrimeagen) [inline src/game/base.ts:187]: Why decrement?

Comment (FL3SH) [reply src/game/base.ts:187]: - to never fulfill the end of game check - endless mode require this
- to have 'knowledge' about round number used for computing average round time

**Pull request merged**